src/ontology/cl-edit.owl

```diff
@@ -27917,11 +27919,25 @@ EquivalentClasses(obo:CL_4023168 ObjectIntersectionOf(obo:CL_0000540 ObjectSomeV
 
 # Class: obo:CL_4023169 (trigeminal neuron)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22190641"^^xsd:string) Annotation(oboInOwl:hasDbXref "WikipediaVersioned:Trigeminal_nerve&oldid=1099897926"^^xsd:string) obo:IAO_0000115 obo:CL_4023169 "A sensory neuron that is responsible for sensation in the face and motor functions such as biting and chewing. Trigeminal neurons extend a single axon shaft along the lateral white matter of the hindbrain and spinal cord. Numerous medially projecting branches are later formed and innervate a series of target nuclei along the anteroposterior axis of the hindbrain and spinal cord. The highly branched axons innervate the integument of the head.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22190641"^^xsd:string) Annotation(oboInOwl:hasDbXref "WikipediaVersioned:Trigeminal_nerve&oldid=1099897926"^^xsd:string) obo:IAO_0000115 obo:CL_4023169 "A neuron that is responsible for sensation in the face or motor functions such as biting and chewing. Trigeminal neurons extend a single axon shaft along the lateral white matter of the hindbrain and spinal cord. The highly branched axons innervate the integument of the head.")
 AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4023169 <http://orcid.org/0000-0001-7258-9596>)
 AnnotationAssertion(rdfs:label obo:CL_4023169 "trigeminal neuron")
-SubClassOf(obo:CL_4023169 obo:CL_4023168)
-SubClassOf(obo:CL_4023169 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0001645))
+SubClassOf(obo:CL_4023169 obo:CL_0000540)
+
+# Class: obo:CL_4023170 (trigeminal sensory neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "WikipediaVersioned:Trigeminal_nerve&oldid=1099897926"^^xsd:string) obo:IAO_0000115 obo:CL_4023170 "A trigeminal neuron that is responsible for sensation in the face.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4023170 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(rdfs:label obo:CL_4023170 "trigeminal sensory neuron")
+EquivalentClasses(obo:CL_4023170 ObjectIntersectionOf(obo:CL_4023169 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0050906)))
+SubClassOf(obo:CL_4023170 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0003942))
+
+# Class: obo:CL_4023171 (trigeminal motor neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "WikipediaVersioned:Trigeminal_nerve&oldid=1099897926"^^xsd:string) obo:IAO_0000115 obo:CL_4023171 "A trigeminal neuron that is responsible for motor functions such as biting and chewing.")
```

Comment:
Not seeing evidence for this cell type with the provided dbxref.

Reply:
If the nerve is capable of multiple function, doesn't it by logic mean that there are neurons in said nerve that can do either of the function

Reply:
If a reference exists that specifically invokes the cell type (e.g., https://www.ncbi.nlm.nih.gov/books/NBK539823/), I think it would be more appropriate to use that for the reference.

"The upper motor neurons commanding lower motor neurons in the motor trigeminal nucleus originate in the motor cortex. They are transmitted bilaterally to the trigeminal motor nuclei in the pons (lateral to the mesencephalic nucleus). They then travel through the middle cerebellar peduncle (directly adjacent to incoming sensory fibers) to emerge from the middle pons to travel in the mandibular division of the CN V.[[2]](https://www.ncbi.nlm.nih.gov/books/NBK539823/#)[[4]](https://www.ncbi.nlm.nih.gov/books/NBK539823/#)

The fifth cranial nerve is related to three sensory nuclei and one motor nucleus related to muscles of mastication and facial sensation. The mesencephalic tract and nucleus, the chief/principal sensory nucleus, spinal trigeminal tract and nucleus, and the trigeminal motor nucleus.[[2]](https://www.ncbi.nlm.nih.gov/books/NBK539823/#)"